lib/versions_repository.py

```diff
@@ -25,3 +25,18 @@ def setup_versions_repository(config):
         raise
 
     return versions_repo
+
+def read_version_and_milestone(versions_repo):
+    """
+    Read current version and milestone (alpha or beta) from VERSION file
+
+    Args:
+        versions_repo (GitRepository): packages metadata git repository
+    """
+    version_path = os.path.join(versions_repo.name, 'VERSION')
+    version_milestone = ""
+    with open(version_path, 'r') as file_version:
+        _ = file_version.readline()
```

Comment:
You dont have to set the function result to a variable. Just write  `file_version.readline()`.